### PR TITLE
[private] #62: Created private office

### LINKED ISSUE #62: Create private office

### PR DESCRIPTION
**Pull request by altarin**
closes #62

### CODE REVIEW

File: makeup/html/private_office.html [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+        </div>
+        <!--личный кабинет-->
+        <div class="section">
+            <div class="title office">ЛИЧНЫЙ КАБИНЕТ</div>
+            <div class="privat">
+                <div class="form">
+                    <div class="my-profile">
+                        <div class="link-office">Мой профиль</div>
+                        <div class=" data data-profile">
+
```
Comment (IvanAlekseevichPopov) [inline makeup/html/private_office.html]: Please remove empty lines

File: makeup/html/private_office.html [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+                            <div class="avatar">
+                                <div class="download">
+                                    <a href="javascript:showFrm()">Загрузить фото</a>
```
Comment (IvanAlekseevichPopov) [inline makeup/html/private_office.html:52]: Is using js here nessesary? Maybe implement it throug label with property 'for'

File: makeup/html/private_office.html [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+                            </table>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+
+    </header>
+
```
Comment (IvanAlekseevichPopov) [inline makeup/html/private_office.html]: Empty lines

File: makeup/html/private_office.html (hunk 2 above)
Comment (altarin) [reply makeup/html/private_office.html:52]: Js need to remain. otherwise, there will be a button, and this is impractical

**Review by IvanAlekseevichPopov — APPROVED** [review]

**Pull request merged**